app/konnect/runtime-manager/gateway-runtime-kubernetes.md

```diff
@@ -0,0 +1,168 @@
+---
+title: Set up a Kong Gateway Runtime on Kubernetes
+no_version: true
+---
+Set up a Kubernetes [runtime](/konnect/overview/#konnect-key-concepts-and-terminology)
+through the
+[{{site.konnect_short_name}} Runtime Manager](/konnect/runtime-manager) and
+configure your {{site.base_gateway}} instance to accept configuration from
+{{site.konnect_short_name}}. The Runtime Manager keeps track of all runtimes
+associated with the {{site.konnect_short_name}} SaaS account.
+
+<div class="alert alert-ee blue">
+<b>Note:</b> Kong does not host runtimes. You must install and host your own
+runtime instances.
+</div>
+
+## Prerequisites
+
+* You have a {{site.konnect_product_name}} account. Contact your sales
+representative for access.
+* **Kubernetes cluster with load balancer:** {{site.konnect_short_name}} is
+compatible with all distributions of Kubernetes. You can use a Minikube, GKE,
+or OpenShift cluster.
+* **kubectl or oc access:** You should have kubectl or oc (if working with OpenShift)
+installed and configured to communicate to your Kubernetes cluster.
+* [Helm 3](https://helm.sh/docs/intro/install/) is installed.
+
+## Set up Helm
+
+On your runtime's system, create a namespace and pull down the `kong` Helm repo.
+
+1. Create a namespace:
+    ```sh
+    $ kubectl create namespace kong
+    ```
+
+2. Add the Kong charts repository:
+    ```bash
+    $ helm repo add kong https://charts.konghq.com
+    ```
+
+3. Update Helm:
+    ```bash
+    $ helm repo update
+    ```
+
+## Generate certificates
+
+{% include /md/konnect/runtime-certs.md %}
+
+## Configure the runtime
+
+Next, configure a {{site.base_gateway}} instance using the certificate, the
+private key, and the remaining configuration details on the
+**Configure Runtime** page.
+
+### Configure secrets
+
+Store the certificates and key you generated through the Runtime Manager in
+Kubernetes secrets.
+
+1. Create a `tls` secret using the `cluster.cert` and `cluster.key` files
+you saved earlier:
+
+    ```bash
+    $ kubectl create secret tls kong-cluster-cert -n kong \
+      --cert=/<path-to-file>/cluster.crt \
+      --key=/<path-to-file>/cluster.key
+    ```
```

Comment:
these files are mapped to tls.key and tls.cert inside the cluster which could cause some confusion. Might help to make these match. As if things aren't working it could cause undue concern.